site/src/sphinx/server-docservice.rst

```diff
@@ -219,3 +219,21 @@ For gRPC:
         </execution>
       </executions>
     </plugin>
+
+For Annotated services:
+
+- Use :api:`@Description` annotation to where you want to describe.
+
+  .. code-block:: java
+
+      import com.linecorp.armeria.server.annotation.Description;
+
+      @Description("A service that provides user information.")
+      public class UserService {
+
+          @Description("get a user by his or her id")
+          @Get("/users/{id}")
```

Comment:
- Could swap the two lines for aesthetics' sake? (Longer line appears later.)
- `get...` -> `Retrieves the user information by the given user ID.` ?